lib/internal/bootstrap/node.js

```diff
@@ -192,13 +192,14 @@ function startup() {
   process.argv[0] = process.execPath;
 
   // Handle `--debug*` deprecation and invalidation.
-  if (process._invalidDebug) {
+  if (!getOptionValue('--inspect') && getOptionValue('--debug')) {
```

Comment:
Regarding @jasnell 's points:

> Unfortunately we can't just remove these with deprecating them first

You can set these properties using on the `process` object using `getOptionValue()` here and then emit deprecation warnings for those.

The warnings here should only be for access to deprecated CLI flags. For access to `process._something` please create new deprecation warnings. See [releases.md](https://github.com/nodejs/node/blob/master/doc/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs) on how those are handled, or you can leave them out of this PR and simply assign those aliases here for this PR.